src/templates/video.js

```diff
@@ -0,0 +1,198 @@
+    mediaType: 'youtube',
+    playlistId: 'PL4RCxklHWZ9sOT_iMTc3RKgSoVbQpe8aX',
+    publishDate: '2020-07-15T14:00:13Z',
+    thumbnailUrl: 'https://i.ytimg.com/vi/sx_hnMUhiHA/sddefault.jpg',
+    title:
+        'Creating a Multiplayer Drawing Game with Phaser and MongoDB, Part 1',
+    videoId: 'sx_hnMUhiHA',
+};
+//
+//Twitch
+// const data = {
+//     description: "",
+//     mediaType: "twitch",
+//     publishDate: "2021-03-31T17:02:32Z",
+//     thumbnailUrl: "https://static-cdn.jtvnw.net/cf_vods/d2nvs31859zcd8/664c72cb8d59f8790fb0_mongodb_42167974830_1617210143//thumb/thumb0-%{width}x%{height}.jpg",
+//     title: "Creating a Pokémon Game with MongoDB Change Streams",
+//     videoId: "970171464",
+// }
+
+const Video = ({
+    pageContext: {
+        data: {
+            description,
+            publishDate,
+            thumbnailUrl: image,
+            title,
+            videoId,
+            mediaType
+        },
+        slug,
+    } = { data, slug: '/test'},
+}) => {
+    const { siteUrl } = useSiteMetadata();
+    const pageUrl = addTrailingSlashIfMissing(`${siteUrl}${slug}`);
+    const formattedPublishDate = toDateString(publishDate, dateFormatOptions);
+
+    const videoBreadcrumb = useMemo(
+        () => [
+            ...VIDEO_BREADCRUMB,
+            {
+                label: `${mediaType} Video`,
+                target: slug,
+            },
+        ],
+        [mediaType, slug]
+    );
+
+    return (
+        <Layout includeCanonical={false}>
+            <SEO
+                articleTitle={title}
+                canonicalUrl={pageUrl}
+                image={image}
+                metaDescription={description}
+                ogDescription={description}
+                ogTitle={title}
+                ogUrl={pageUrl}
+                twitter={{
+                    description,
+                    image,
+                    title,
+                }}
+            />
+            <ArticleSchema
+                articleUrl={pageUrl}
+                title={title}
+                description={description}
+                publishedDate={formattedPublishDate}
+                imageUrl={image}
+            />
+            <VideoJumbotron
+                breadcrumb={videoBreadcrumb}
+                publishDate={formattedPublishDate}
+                title={title}
+            />
+            <Container>
+                <Icons>
+                    <ShareMenu
+                        height={size.default}
+                        title={title}
+                        url={pageUrl}
+                        width={size.default}
+                    />
+                </Icons>
+                <Content>
+                    <div>
```

Comment:
Is this `div` doing something here or could it be removed?

Reply:
Nothing, it's just forgotten tag, I will remove it.